### PR TITLE
Setup component

### PR DESCRIPTION
**Pull request by hiromataba**
Style Wireframe of the calculator

### CODE REVIEW

File: src/components/calculator.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          <button className="btn" type="submit">4</button>
+          <button className="btn" type="submit">5</button>
+          <button className="btn" type="submit">6</button>
+          <button className="btn" type="submit">&#8722;</button>
+        </div>
+        <div className="row-flex">
+          <button className="btn" type="submit">1</button>
+          <button className="btn" type="submit">2</button>
+          <button className="btn" type="submit">3</button>
+          <button className="btn" type="submit">&#215;</button>
```
Comment (Dipeshtwis) [inline src/components/calculator.jsx]: - [ ] Please make sure to use the correct symbol for each place and avoid duplication. Currently, you have no `%` symbol. Also, there is two `multiplication` sign. Please fix it.

### Your design

Screenshot from 2021-08-10 20-45-07

### Given design

Screenshot from 2021-08-10 20-48-42

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @hiromataba, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Linters are passing :white_check_mark: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

- [ ] You have removed almost all the file which was not required, but still, some of them are left, Please Remove `App.test.js`, `setupTests.js`, and `.hintrc`, as well as we, are not testing anything for this moment and we are not using `HTML` linters.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+          <button className="btn" type="submit">4</button>
+          <button className="btn" type="submit">5</button>
+          <button className="btn" type="submit">6</button>
+          <button className="btn" type="submit">&#8722;</button>
+        </div>
+        <div className="row-flex">
+          <button className="btn" type="submit">1</button>
+          <button className="btn" type="submit">2</button>
+          <button className="btn" type="submit">3</button>
+          <button className="btn" type="submit">&#247;</button>
```
Comment (Dipeshtwis) [inline src/components/calculator.jsx]: - [ ] I think you have interchanged the position of `+` and `÷` text. Please fix it to make the design similar.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @hiromataba , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**

-------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.jsx (hunk 2 above)
Comment (DeliceLydia) [reply src/components/calculator.jsx]: Well addressed that!

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉

## Hi @hiromataba, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**